code/modules/cargo/packs.dm

```diff
@@ -543,6 +543,14 @@
 	crate_name = "electrical maintenance crate"
 	crate_type = /obj/structure/closet/crate/engineering/electrical
 
+/datum/supply_pack/engineering/inducers
+	name = "NT-75 Electromagnetic Power Inducers Crate"
+	cost = 2000
+	contains = list(/obj/item/weapon/inducer,
+					/obj/item/weapon/inducer)
```

Comment:
Literallly every entry in that file splits the line if the crate holds more than one item.